tools/capture_radar.rb

```diff
@@ -0,0 +1,67 @@
+Metric
+class Metric
+  def derived_cpu_total_cores_used
```

Comment:
@Ladas Would you be able to confirm whether this calculation for used CPU cores is correct?

Reply:
@yaacov can you confirm this is correct?  We store total numbet of Container's cores in ```derived_vm_numvcpus```, right? So in that case the calculation should be correct. Too bad we didn't merge the PR with storing used cores directly.

@yaacov have we backported the PRs fixing CPU usage incorrect values to Fine? If not, can we expect non accurate CPU util values?

Reply:
> We store total numbet of Container's cores in derived_vm_numvcpus, right?

yes, during processing we use 'vim_performance_state_for_ts' to set it: 
https://github.com/ManageIQ/manageiq/blob/master/app/models/metric/processing.rb#L48
https://github.com/ManageIQ/manageiq/blob/master/app/models/metric/processing.rb#L105

> have we backported the PRs fixing CPU usage incorrect values to Fine? If not, can we expect non accurate CPU util values?

https://github.com/ManageIQ/manageiq-providers-kubernetes/pull/159 is G backported not F